src/test/scala/io.circe.config/CirceConfigSpec.scala

```diff
@@ -38,9 +38,9 @@ class CirceConfigSpec extends AnyFlatSpec with Matchers {
 
     val config = decode
```

Comment:
I'd be tempted just to `val Right(config) = decode` this one too, since it's a test, and since the failure wouldn't be worse and might arguably be a little better.

Reply:
True. Will do that once I figure out why the CI runs failed.